metadata-ingestion/tests/integration/looker/test_looker.py

```diff
@@ -285,10 +286,11 @@ def setup_mock_dashboard(mocked_client):
         created_at=datetime.utcfromtimestamp(time.time()),
         updated_at=datetime.utcfromtimestamp(time.time()),
         description="lorem ipsum",
+        folder=FolderBase(name="Shared", id="shared-folder-id"),
         dashboard_elements=[
             DashboardElement(
                 id="2",
-                type="",
+                type="vis",
```

Comment:
Required due to this [check](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/looker/looker_source.py#L778), otherwise the charts don't get ingested